src/pages/projects/[index].tsx

```diff
@@ -1,197 +1,65 @@
-                label: 'https://www.motherhoodreclaimed.org/',
-                url: 'https://www.motherhoodreclaimed.org/',
-            },
-            {
-                label: 'https://github.com/Open-SGF/motherhood-reclaimed-website',
-                url: 'https://github.com/Open-SGF/motherhood-reclaimed-website',
-            },
-        ],
-        toolsUsed: [
-            {
-                label: 'CraftCMS',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-            {
-                label: 'Node',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-            {
-                label: 'MySQL',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-        ],
-        contributors: [
-            {
-                name: 'Ethan Zitting',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Levi Zitting',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Shannon Treadway',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Aaron Merrick',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-        ],
-        clientIssue: 'Client needed a website to help share their mission to' + ' potential partners',
-        ourSolution:
-            'We worked with them to come up with a design they liked and' +
-            ' then launched their website with a CMS so they could edit the' +
-            ' content as needed.',
-    };
-
-    const router = useRouter();
 
+export const getServerSideProps: GetServerSideProps = async ({ query: { index: url } }) => {
+    let contributors;
     let projectData;
 
-    if (router.asPath.includes('mo-jobs')) {
+    if (url === 'mo-jobs') {
+        const apiContributors = await getProjectContributors('portal-to-work-api');
+        const clientContributors = await getProjectContributors('portal-to-work-client');
```

Comment:
Hard coding this is not great. There really no way to map a url to content in such a way this function doesn't need to be updated forevery project we add?

Reply:
The nice thing about async functions is that they can be chained together to be called at the same time. Here we have two back-to-back awaits which means that before the second await gets called, we have to wait for the first one. These two async calls are mutually exclusive, and therefore can be called in parallel: https://stackoverflow.com/a/35612484